lighthouse-core/audits/redirects-http.js

```diff
@@ -6,6 +6,19 @@
 'use strict';
 
 const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on HTTP to HTTPS redirects. This descriptive title is shown to users when HTTP traffic is redirected to HTTPS. */
+  title: 'Redirects HTTP traffic to HTTPS',
+  /** Title of a Lighthouse audit that provides detail on HTTP to HTTPS redirects. This descriptive title is shown to users when HTTP traffic is not redirected to HTTPS. */
+  failureTitle: 'Does not redirect HTTP traffic to HTTPS',
+  /** Description of a Lighthouse audit that tells the user why they should direct HTTP traffic to HTTPS. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'If you\'ve already set up HTTPS, make sure that you redirect all HTTP ' +
+    'traffic to HTTPS in order enable secure web features for all your users. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/http-redirects-to-https).',
```

Comment:
I seem to remember saying something like this. It seems to me that *in the context of PWA* the primary reason you care about this is for using a service worker for all your visitors. That's a very concrete and specific carrot compared to "make it secure so that it will be secure" but 🤷‍♂

Reply:
> I seem to remember saying something like this. It seems to me that _in the context of PWA_ the primary reason you care about this is for using a service worker for all your visitors. That's a very concrete and specific carrot compared to "make it secure so that it will be secure" but 🤷‍♂

well the most important reason to make it secure *is* so it will be secure :D but very good point about the context and the whole point is to be checking something a dev might forgot to check that will undermine other things they're trying to do. So change SGTM, but still needs the typo fix

```suggestion
    'traffic to HTTPS in order to enable secure web features for all your users. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/http-redirects-to-https).',
```